platformio.ini

```diff
@@ -0,0 +1,22 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:esp32dev]
```

Comment:
It may be better to have common options which can easily be re-used for any supported board, e.g.

```
[env]
platform = espressif32
framework = arduino
monitor_speed = 115200
lib_deps =
	ArduinoJson @ 7.0.3
	MQTT @ 2.5.2

[env:esp32dev]
board = esp32dev

[env:lolin_s2_mini]
board = lolin_s2_mini
```

Reply:
I've applied the comments. I don't have windows, so I referred window users to the docs.